cylc/flow/job_file.py

```diff
@@ -149,6 +151,22 @@ def _write_directives(self, handle, job_conf):
             for line in lines:
                 handle.write('\n' + line)
 
+    def _write_reinvokcation(self, handle):
```

Comment:
```suggestion
    def _write_reinvocation(self, handle):
```

Reply:
Ping @oliver-sanders - looks like you missed this companion suggestion. (Expecting tests to all fail!)